### PR TITLE
docs: roll 1.1.0

### PR DESCRIPTION
**Pull request by andreivladbrg**
~~This PR https://github.com/sablier-labs/v2-core/pull/683 is still not merged/closed. We may add the changes here.~~

### CODE REVIEW

**Comment by PaulRBerg** [comment]
Could you please create a new PR that adds the changelog to the `2.1` branch? Once we merge that in, we can then merge everything on `main`

**Comment by andreivladbrg** [comment]
>Could you please create a new PR that adds the changelog to the 2.1 branch?  

Update this PR base branch to `2.1`

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,10 +4,26 @@ All notable changes to this project will be documented in this file.
...
+[1.1.0]: https://github.com/sablier-labs/v2-core/compare/v1.0.2...v1.1.0
 [1.0.2]: https://github.com/sablier-labs/v2-core/compare/v1.0.1...v1.0.2
 [1.0.1]: https://github.com/sablier-labs/v2-core/compare/v1.0.0...v1.0.1
 [1.0.0]: https://github.com/sablier-labs/v2-core/releases/tag/v1.0.0
 
+## [1.1.0] - 2023-09-26
+
+### Changed
+
+- Change the streamed amount card in NFT descriptor with deposit amount
```
Comment (PaulRBerg) [inline CHANGELOG.md:16]: Let's say:

"Replace the streamed amount with the deposit amount in the NFT descriptor"

**Review by PaulRBerg — CHANGES_REQUESTED** [review]
Left two comments below.

Besides those, we also need to address https://github.com/sablier-labs/v2-core/issues/635 before merging this PR.

Opening a PR now.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,10 +4,26 @@ All notable changes to this project will be documented in this file.
...
+  ([692](https://github.com/sablier-labs/v2-core/pull/692))
+- Upgrade Solidity to `0.8.21` ([688](https://github.com/sablier-labs/v2-core/pull/688))
```
[the remark this answers is not in the document]
Comment (andreivladbrg) [reply CHANGELOG.md:18]: Yep, typo

**Pull request merged**